backend/ormconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "sqlite",
```

Comment:
I guess this is just a temporal solution.

Reply:
obviously - the package is not even installed